tests/e2e_test.go

```diff
@@ -35,6 +35,24 @@ var (
 )
 
 func TestE2E(t *testing.T) {
+	// In case of a panic we need to recover to ensure the Ginkgo cleanup is done
+	defer func() {
```

Comment:
I don't think we actually need this. It looks like this would only catch top level panics, and those do seem to be handled by Ginkgo. 

Issues we had were with panics in goroutines but this doesn't catch those.